### ISSUE TITLE
What is the correct format to set "UTC offset" timezone for NSG Location?

### ISSUE THREAD

**Issue by hellt** [nuagenetworks/vsd-api-documentation#3]
As per VSD API Doc for [Location](https://nuagenetworks.github.io/vsd-api-documentation/v4_0/location.html#attr_timeZoneID) object we can use both UTC/GMT offset notation for timezones.

While I can manage to set GMT offset value in this way `GMT+03:00` I cant succeed with UTC offset passed in the same manner. 

![image](https://cloud.githubusercontent.com/assets/5679861/24703773/6b27d988-1a0d-11e7-9427-1c1304f26a90.png)

If the GMT is the only way to set offsets, should we reflect this in the parameter description, or give an example how to pass offsets for both variants?

**Comment by pdellaert** [comment]
Best way of figuring this out is to open a VSD and add `?debug` at the end (for instance: `https://10.189.1.60:8443?debug`). When opening the developer JS console, you'll see the exact API calls being made. That way you can configure the above information from the UI, see what API call is being made, and deduce the timeZoneID field from that call.

**Comment by hellt** [comment]
Right, but the question was "How to initially compose the string which will set UTC offset?" 
If the only offset format available with is the one that goes as `GMT+hh:mm` then maybe it will be nice to clarify this in a doc.

(Since `UTC+hh:mm` is not compatible with VSD API)

**Comment by pdellaert** [comment]
Have you tried with the actual location information, for instance Europe/Brussels
(list: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)

**Comment by hellt** [comment]
This works like a charm, of course, we do it all the time.
But if one need to specify an UTC offset and not the actual name of the TZ,
how would one do it?

Based on the article you mentioned I should try +03; +03:00 as well. Maybe
this will be considered like UTC+3 offset

5 апр. 2017 г. 5:22 PM пользователь "Philippe Dellaert" <
notifications@github.com> написал:

**Comment by pdellaert** [comment]
Not sure what exactly works, but i'm just looking at what the documentation says :). Not sure if UTC values will work, but it's worth a try

**Comment by hellt** [comment]
Checked other variants of passing UTC offset. None of them worked.
So to sum it up:
Citing VSD API Docs for Location obj:
>Time zone in which the Gateway is located. This can be in the form of a UTC/GMT offset, continent/city location, or country/region.  

Not entirely correct. I could not find a way to pass a UTC offset by means of `UTC+<offset>`
